lib/WIFI/devWIFI.cpp

```diff
@@ -250,9 +253,14 @@ static void WebUpdateSetHome(AsyncWebServerRequest *request)
   String password = request->arg("password");
 
   DBGLN("Setting home network %s", ssid.c_str());
-  config.SetSSID(ssid.c_str());
-  config.SetPassword(password.c_str());
-  config.Commit();
+  strcpy(station_ssid, ssid.c_str());
+  strcpy(station_password, password.c_str());
+  // Only save to config if we don't have a flashed wifi network
+  if (home_wifi_ssid[0] == 0) {
```

Comment:
What does `0` represent here? no saved creds on flash? Might be nice to make this a define so it's a bit clearer what we're checking for?

Reply:
0 is the NUL character which is the null-terminator for a C-style string.
It would be nice if there was a canonical way of doing a empty check but theres not really.